mysql/datadog_checks/mysql/config.py

```diff
@@ -52,6 +52,8 @@ def __init__(self, instance):
                     'replace_digits', obfuscator_options_config.get('quantize_sql_tables', False)
                 )
             ),
+            'dollar_quoted_func': is_affirmative(obfuscator_options_config.get('dollar_quoted_func', False)),
```

Comment:
non-blocker: The key in this dict should match what the obfuscator expects, but what do you think about making this public facing option a little more descriptive, maybe `keep_dollar_quoted_func`?

Reply:
I definitely agree that the name is no good, what do you think about `obfuscate_within_dollar_quoted_func`?  I'm pretty sure what this option does is assume that anything inside of a string like:
```
$func$
select * from blah where x = 10
$func$
```
is a sql query, and will recursively call the obfuscate on it, yielding:
$func$
select * from blah where x = ?
$func$

https://github.com/DataDog/datadog-agent/blob/main/pkg/obfuscate/sql_tokenizer.go#L387-L396

Reply:
Although maybe `keep_` is good enough and I'm just over thinking it.

Reply:
Yeah, `keep_dollar_quoted_func` is concise and to the point, so lgtm.

Your explanation brought up a nice idea, should we add a basic example in the config? Initially I had no idea what it was doing, so I think it would be helpful.

Reply:
Yeah, will do